lib/datadog/appsec/processor.rb

```diff
@@ -70,7 +70,11 @@ def create_waf_handle
         # TODO: this may need to be reset if the main Datadog logging level changes after initialization
         Datadog::AppSec::WAF.logger = Datadog.logger if Datadog.logger.debug? && Datadog::AppSec.settings.waf_debug
 
-        @handle = Datadog::AppSec::WAF::Handle.new(@ruleset)
+        obfuscator = {
```

Comment:
Minor: Maybe call this `obfuscator_config`?

Reply:
Addressed